src/stores/timed.rs

```diff
@@ -1,9 +1,12 @@
+use instant::Instant;
```

Comment:
It should be removed

Reply:
don`t see what should be removed, can you elaborate?

Reply:
This use of `Instant`. Or is it in user somewhere?

Reply:
It is used for the store on line 32
```rs
pub struct TimedCache<K, V> {
    pub(super) store: HashMap<K, (Instant, V)>,
    pub(super) seconds: u64,
    pub(super) hits: u64,
    pub(super) misses: u64,
    pub(super) initial_capacity: Option<usize>,
    pub(super) refresh: bool,
}
```